index.md

```diff
@@ -195,96 +129,113 @@ either the result of a digital migration or information that enables the creatio
 of an emulation environment to render a representation. The result of this
 operation is the creation of a new version of the AIP.
 
-The result of a new version of the AIP is stored separately from the submission
-as explained in detail in section [5.2.2](#aipreps).
-
-Furthermore, the AIP format allows updating the AIP by adding a new version of
-the submission. This allows supporting the AIP edition which is defined in OAIS
-as follows:
+Furthermore, the AIP format allows updating the AIP which correspond to the AIP 
+edition which is defined in OAIS as follows:
 
 > “AIP  Edition: An  AIP  whose  Content  Information  or Preservation
 > Description  Information  has been upgraded or improved with the intent not to
 > preserve information, but to increase or improve it. An AIP edition is not
 > considered to be the result of a Migration. [@OAIS2012, p. 1-9]”
 
```

Comment:
This is interesting and is also at odds with my use of "generation". I'm not sure I've quite crisply comprehended the criteria that differentiates an edition from a version.

Reply:
My understanding is that here they make a difference between migrating content files where the purpose is to maintain the (significant) properties of the derivative as close as possible to the original, whereas the edition is a change of content or metadata with the purpose to improve or extend the information package. Our definition of "generation" refers to the physical container manifestation. The "edition" concept does not.